src/utils/blocked-uas.ts

```diff
@@ -48,9 +48,9 @@ export const DEFAULT_BLOCKED_UA_STRS = [
     'storebot-google',
 ]
 
-// _.isBlockedUA()
-// This is to block various web spiders from executing our JS and
-// sending false capturing data
+/**
```

Comment:
Formatting comments with the `/**` syntax on functions & type declarations makes it so VSCode will show the comments inline on hover.

See https://code.visualstudio.com/docs/languages/typescript#_hover-information for more info

![Screenshot 2024-01-05 at 3 55 18 AM](https://github.com/PostHog/posthog-js/assets/127199/ca9e97b1-1d89-44b6-b412-20fb0133bb11)

Reply:
We were just talking about this yesterday. I use intellij based IDEs and that shows either kind... It's the new tabs or spaces 🤣 text editor or IDE 🤣

Reply:
(more seriously - I didn't know it didn't work that way for all and agree that if the comment is useful we should show it in as many places as possible - and if it isn't we should delete it :))